src/components/organisms/modals/UserAndRoleManagementModal/UserAndRoleManagementModal.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, ReactElement } from 'react'
+import { AppearanceTypes, SizeTypes } from 'components/molecules/Button/Button'
+import ModalBase, {
+  ButtonPositionTypes,
+  TitleFontTypes,
+} from 'components/organisms/ModalBase/ModalBase'
+
+export interface UserAndRoleManagementModalProps {
+  title?: string
+  cancelButtonContent?: string
+  proceedButtonContent?: string
+  modalContent?: string | ReactElement
+  isModalOpen?: boolean
+  closeModal: () => void
+  handleProceed?: () => void
+  className?: string
+}
+
+const UserAndRoleManagementModal: FC<UserAndRoleManagementModalProps> = ({
+  title,
+  cancelButtonContent,
```

Comment:
**[blocker]**

Tend to agree with Maret here, that it would be good to at least have "yes" and "no" as the default values for proceedButtonContent and cancelButtonContent. Didn't go over all of the design, but I'm not sure if we will have any cases, where they are anything other than "yes" and "no" for confirmation modals.

Especially if this is not a generic confirmation modal, but rather something that is just used for "UserAndRoleManagement" as the name suggests.